music/src/core/logging.ts

```diff
@@ -15,23 +15,34 @@
  * limitations under the License.
  */
 
-import {performance} from '../core/compat/global';
+import { performance } from '../core/compat/global';
 
 /**
  * The different verbosity levels.
  */
 export const enum Level {
-  NONE = 0,   // No messages will be logged.
-  WARN = 5,   // WARN messages will be logged.
-  INFO = 10,  // INFO and WARN messages will be logged.
-  DEBUG = 20  // DEBUG, INFO, and WARN messages will be logged.
+  NONE = 0, // No messages will be logged.
```

Comment:
This code formatting change seems strictly worse, for example

Reply:
i do have prettier setup and personally have always preferred the consistent formatting that prettier gives me and my team. i wasn't aware that the repo itself has another formatter since it never kicked in for me even when i disabled prettier. let me see whether i can setup clang and resave these files.